### PR TITLE
Note about interpolations with null values

### PR DESCRIPTION
**Pull request by TomSeymour**

### CODE REVIEW

File: Documentation/SequentialDataStore/Reading_Data_API.md [hunk 1]
```diff
@@ -1410,6 +1410,7 @@ SDS supports the following types of joins:
 | MergeLeft    | 3                 | Results include events for each index across all streams selecting events at the indexes based on left to right order of the streams. |
 | MergeRight   | 4                 | Results include events for each index across all streams selecting events at the indexes based on right to left order of the streams. |
 
+Note that the Interpolated SdsJoinMode currently does not support ContinuousNullableTrailing or ContinuousNullableLeading SdsInterpolationModes. All join requests with interpolations involving a Null value will return an interpolated Null result at the index where calculation was required.
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Reading_Data_API.md]: Change "Note that" to **Note:** The Interpolated SdsJoinMode.....
(** before and after "Note" is for bolding)

File: Documentation/SequentialDataStore/Reading_Data_API.md [hunk 2]
```diff
@@ -1410,6 +1410,7 @@ SDS supports the following types of joins:
 | MergeLeft    | 3                 | Results include events for each index across all streams selecting events at the indexes based on left to right order of the streams. |
 | MergeRight   | 4                 | Results include events for each index across all streams selecting events at the indexes based on right to left order of the streams. |
 
+**Note:** The Interpolated SdsJoinMode currently does not support ContinuousNullableTrailing or ContinuousNullableLeading SdsInterpolationModes. All join requests with interpolations involving a Null value will return an interpolated Null result at the index where calculation was required.
```
Comment (mckenzielee) [inline Documentation/SequentialDataStore/Reading_Data_API.md]: Should this be in the content that is specific to Interpolated Joins? Also, we should provide a link to the interpolation and extrapolation content in the Interpolated Joins section.

Comment (TomSeymour) [reply Documentation/SequentialDataStore/Reading_Data_API.md]: I added a link and moved the text.

**Comment by mckenzielee** [comment]
@osimarkrogers, I am merging this change, since it represents current functionality available in the Production.

**Pull request merged**